waspc/data/Generator/templates/universal/url.ts

```diff
@@ -0,0 +1,3 @@
+export function stripTrailingSlash(url?: string): string | undefined {
```

Comment:
**Nit:** If I'm not mistaken, the signature is a bit underspecified. We have:
```typescript
(x: string | undefined) => string | undefined
```
While the "correct" signature would be:
```typescript
(x: string) => string | (x: undefined) => undefined
```

Reply:
We agreed offline that we are okay with the current signature 👍